### PR TITLE
Add JWT to Developer view in tenant UI

### PR DESCRIPTION
**Pull request by Gavinok**
Added support for viewing encoded and decoded JWTs as well as quickly copying them to clipboard resolving #611.

image

When the copy button is clicked a notification is displayed to inform them

image

### CODE REVIEW

File: services/tenant-ui/frontend/src/components/profile/Developer.vue [hunk 1]
```diff
@@ -2,14 +2,67 @@
   <!-- eslint-disable-next-line @intlify/vue-i18n/no-raw-text -->
   <h2>{{ $t('profile.config') }}</h2>
   <vue-json-pretty :data="config" />
+  <h2>{{ $t('profile.token') }}</h2>
```
Comment (amanji) [inline services/tenant-ui/frontend/src/components/profile/Developer.vue:5]: Good practice moving forward is to maybe encapsulate these into their own components

**Review by amanji — APPROVED** [review]

**Comment by amanji** [comment]
You'll need to update your branch before it can be merged

**Review by amanji — DISMISSED** [review]

File: services/tenant-ui/frontend/src/components/profile/JWT.vue [hunk 2]
```diff
@@ -0,0 +1,60 @@
+<template>
+  <h2>{{ $t('profile.token') }}</h2>
+  <Accordion :multiple="true">
+    <AccordionTab header="Encoded JWT">
+      <div style="display: flex">
+        <div style="overflow: auto; width: 100%">
+          <!-- Just used p-inputtext to keep styling consistent -->
+          <div class="p-inputtext" style="overflow: auto">
```
Comment (loneil) [inline services/tenant-ui/frontend/src/components/profile/JWT.vue:8]: Should stay away from inline styles as much as we can. App uses https://www.primefaces.org/primeflex/ which provides framework classes for these if needed 

https://www.primefaces.org/primeflex/flex
https://www.primefaces.org/primeflex/overflow
https://www.primefaces.org/primeflex/width

(or apply with a named class in the style block of the component or something).

**Review by loneil — APPROVED** [review]
Great stuff, will be very helpful for debugging/advanced usage with swagger

**Pull request merged**